### PR TITLE
Refactoring part's of the code

### PR DESCRIPTION
**Pull request by NabsiDev**
First of all i replaced all of the string asdf shitty things with right code second i removed  all junk code which isnt needed it was too messy then i recoded / removed unnecessary and also made register system bit better and i guess thats it i hope you guys like it

### CODE REVIEW

**Comment by playingoDEERUX** [comment]
also add stoi() conversion with try catch to places where server interacts with client and retrieves an integer which is much much safer than atoi

**Comment by playingoDEERUX** [comment]
(tried to pr that one but I guess accidentally put a character somewhere and it failed and I guess 1 single merge is better)

**Comment by NabsiDev** [comment]
@DrOreo002 of course but i guess most of people alrdy know what they do so why u would keep them there without reason but thanks for critism

**Comment by NabsiDev** [comment]
and im pretty sure that no1 would need them anyway thats why i deleted them they are only player states.

**Comment by NubVann** [comment]
Omg the scammer is back

**Comment by DrOreo002** [comment]
@NabsiDev well this isn't just a simple program, most of the information / function might are reverse engineered from actual growtopia or retrieved from gt using proxy. Shouldn't be better to keep it so other people that want to make a private server have extra information for them to start off?

**Comment by NabsiDev** [comment]
oh yeah that might be true ill add them back after ill solve how the fuck ima fix that 1 error in the compiling its not my fault its the bcrypt fault idk why it causes that

File: enet server test/enet server test.cpp [hunk 1]
```diff
@@ -2628,14 +2557,13 @@ void loadnews() {
 				data.netID = netID;
 				data.plantingTree = state;
 				BYTE* raw = packPlayerMoving(&data);
-				int var = 0x808000; // placing and breking
```
Comment (DrOreo002) [inline enet server test/enet server test.cpp:2631]: This for example. How other will know whats the meaning of `0x808000` if you removed the comments?

Comment (NabsiDev) [reply enet server test/enet server test.cpp:2631]: well i got a good idea to solve that

Comment (NabsiDev) [reply enet server test/enet server test.cpp:2631]: sec

**Comment by ama6nen** [comment]
fk its not my fault its bcrypt and fkin comments making it messy :((((((((((((((((((((

**Comment by iProgramMC** [comment]
> fk its not my fault its bcrypt and fkin comments making it messy :((((((((((((((((((((

lol stop watching gt issues and go work on ur server cuz there are indogs waiting for u to finish the next performance test

**Pull request closed without merging**